torch/onnx/utils.py

```diff
@@ -1660,6 +1661,50 @@ def _export(
     return torch_out
 
 
+@_beartype.beartype
+def _add_onnxscript_fn(
+    model_bytes: bytes, graph: _C.Graph, custom_opsets: Mapping[str, int]
+) -> bytes:
+    """Insert model-included custom onnx-script function into ModelProto"""
+
+    # TODO(titaiwang): remove this when onnx becomes dependency
+    try:
+        import onnx
+    except Exception:
```

Comment:
```suggestion
    except ImportError:
```

I would avoid catching too generic errors, as errors like MemoryError, KeyboardInterrupt etc. should still need to manifest as themselves.

Reply:
Done